server/modules/modules.go

```diff
@@ -116,3 +117,207 @@ func Auth(message *messageService.Message, conn *websocket.Conn) {
+				Type_:   coremessage.DeleteContactCommand,
+				Command: coremessage.StatusOk,
+			},
+			Body: nil,
+		})
+	conn.WriteMessage(messageType, byteSuccess)
+	return
+}
+
+func DeleteUser(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	var deleteContact *contactService.Contacts
+	var err error
+	deleteContact, err = contactService.UnmarshalContacts(*message)
+
+	if _, err = contactService.RemoveUser(deleteContact.ContactUser); err != nil {
+		loger.Log.Errorf("deleting user from contacts, users and authintications tables failed server error: ", err)
+		byteError, _ := json.Marshal(
+				messageService.Message{
+				Header:
+				messageService.MessageHeader{
+					Type_:   coremessage.DeleteUserCommand,
+					Command: "User was not deleted",
+				},
+				Body: nil,
+			})
+		conn.WriteMessage(messageType, byteError)
+		return
+	}
+
+	byteSuccess, _ := json.Marshal(
+		messageService.Message{
+			Header:
+			messageService.MessageHeader{
+				Type_:   coremessage.DeleteUserCommand,
+				Command: coremessage.StatusOk,
+			},
+			Body: nil,
+		})
+	conn.WriteMessage(messageType, byteSuccess)
+	return
+}
+
+func ShowUserInfo(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	var userInfo *contactService.Users
+	var err error
+	userInfo, err = contactService.UnmarshalUsers(*message)
+
+	if _, err = contactService.GetUserInfo(userInfo.UserName); err != nil {
+		loger.Log.Errorf("selecting user information from users tables failed server error: ", err)
+		byteError, _ := json.Marshal(
+			messageService.Message{
+				Header:
+				messageService.MessageHeader{
+					Type_:   coremessage.ContactType,
+					Command: coremessage.ShowUserInfoCommand,
+				},
+				Body: nil,
+			})
+		conn.WriteMessage(messageType, byteError)
+		return
+	}
+
+	byteVal, _ := json.Marshal(userInfo)
+	body, _ := json.RawMessage.MarshalJSON(byteVal)
+	byteSuccess, _ := json.Marshal(
+		messageService.Message{
+			Header:
+			messageService.MessageHeader{
+				Type_:   coremessage.ContactType,
+				Command: coremessage.ShowUserInfoCommand,
+			},
+			Body: body,
+		})
+	conn.WriteMessage(messageType, byteSuccess)
+	return
+}
+
+func ShowAllContacts(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	var userInfo *contactService.Users
+	var userContacts []contactService.Users
+	var err error
+	userInfo, err = contactService.UnmarshalUsers(*message)
+
+	if userContacts, err = contactService.GetContacts(userInfo.UserName); err != nil {
+		loger.Log.Errorf("selecting all contacts failed server error: ", err)
+		byteError, _ := json.Marshal(
+			messageService.Message{
+				Header:
+				messageService.MessageHeader{
+					Type_:   coremessage.ContactType,
+					Command: coremessage.ShowAllContactsCommand,
+				},
+				Body: nil,
+			})
+		conn.WriteMessage(messageType, byteError)
+		return
+	}
+
+	byteVal, _ := json.Marshal(userContacts)
+	body, _ := json.RawMessage.MarshalJSON(byteVal)
```

Comment:
You don't have to do this call. Simply assign `byteVal` to `Message.Body`. `json.RawMessage` is `[]byte` under the hood.